### ISSUE TITLE
owntracks-recorder docker - nothing is getting recorded

### ISSUE THREAD

**Issue by cannfoddr** [owntracks/recorder#223]
I have a owntracks-recorder docker runnning with 2 clients talking to home-assistant.  Clients are updating and HA is getting to see the updates.

However the recorder web ui shows nothing.

If I run ocat --list from within the container i get:

`{"error":"Cannot` open requested directory"}

I am sure I am doing something stupid but cannot see where to start

**Comment by jpmens** [comment]
How about starting from the beginning, telling us what platform you're working on, how you installed it, what errors you see where, etc. ? It's a bit difficult to start guessing

**Comment by cannfoddr** [comment]
Thanks for the reply.....its the official docker image running on an ubuntu host.  I figured out the issue - as usual was pretty obvious once I started digging.  I had implemented a password policy on mosquito for users and Homessistant but had not provided a mechanism for owntracks-recorder to login to mosquito.

I added a user account and password to the owntracks-recorder config and everything is working fine.
